Extensions/accesskit.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE AccessKit **//
-//* VERSION 1.1 REV D **//
+//* VERSION 1.1.5 **//
```

Comment:
Is there any reason for the large bump in patch version? This might be a nitpick, but I'd prefer either 1.1.0 or 1.1.1.